roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -8,11 +8,33 @@
 from openshift_checks import OpenShiftCheck, OpenShiftCheckException
 
 
+class MissingComponentPods(OpenShiftCheckException):
+    """Raised when a component has no pods in the namespace."""
+    pass
+
+
+class CouldNotUseOc(OpenShiftCheckException):
+    """Raised when ocutil has a failure running oc."""
+    pass
+
+
+class LoggingErrorList(OpenShiftCheckException):
```

Comment:
I wasn't sure if it would be useful anywhere else. I'm not sure any other checks have a need to compile a list of failures instead of just quitting on the first. But again, thinking about it... it will probably come up. I guess I'll generalize it.